### PR TITLE
Fixing typos in the guide and webpage

### PR DESCRIPTION
**Pull request by alanazar**
This fixes typos in the guide and webpage and corrects some language

### CODE REVIEW

File: docs/general/server/users/adding-managing-users.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ To manage a user either click on their portrait to go straight to their `Profile
 
 Directly under the tabs you have a link to `Edit this user's profile, image and personal preferences.` Clicking that allows you to change the user's personal settings, any setting here can be changed by both user and admin.
 
-Under `name` you can change the user's name as it will be either displayed on, or has to be typed into login screen.
+Under `name` you can change the user's name as it will be either displayed on or has to be typed into login screen.
```
Comment (dkanada) [inline docs/general/server/users/adding-managing-users.md:26]: this doesn't seem like an improvement

File: docs/general/faq.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ We explain our rationale on our [about page](/docs/general/about).
 
 ## Why don't you support my favorite client or feature?
 
-Chances are, we'd like to support it! But it hasn't been implemented yet.
-Jellyfin is an entirely volunteer-driven project, so until a developer is able and willing to implement a feature, it likely won't be done.
+Chances are, we would like to support it! But it has not been implemented yet.
+Jellyfin is an entirely volunteer-driven project, so until a developer is able and willing to implement a feature, it likely will not be done.
```
Comment (dkanada) [inline docs/general/faq.md:16]: Were these contraction removals done with an automated tool? And is there a particular reason you chose to remove them?

Comment (alanazar) [reply docs/general/faq.md:16]: > Were these contraction removals done with an automated tool? And is there a particular reason you chose to remove them?

No I manually changed them throughout, it's more formal this way

File: docs/general/server/users/adding-managing-users.md (hunk 1 above)
Comment (alanazar) [reply docs/general/server/users/adding-managing-users.md:26]: Grammar check recommended it

File: docs/general/faq.md (hunk 2 above)
Comment (dkanada) [reply docs/general/faq.md:16]: figured it might be something like that and I generally agree - sounds good to me

**Comment by dkanada** [comment]
Lots of great fixes in here! Do you plan on making further changes or can this be considered ready for review?

**Comment by alanazar** [comment]
> Lots of great fixes in here! Do you plan on making further changes or can this be considered ready for review?

Thank you! :)
I think it'll be a while before I can review this again :D so yeah ready for review

File: docs/general/faq.md (hunk 2 above)
Comment (alanazar) [reply docs/general/faq.md:16]: Thanks!

**Comment by alanazar** [comment]
@dkanada made a few more modifications on this and now I'm getting these weird errors that seem to be a false-positive.

**Comment by alanazar** [comment]
Someone please help with the last error. Not able to solve it, thanks!

**Comment by dkanada** [comment]
@thornbill might know why those escaped whitespaces are used (not quite sure) but the error just looks like a max line length rule so you can fix it by making sure the line doesn't exceed a certain number of characters.

**Comment by alanazar** [comment]
> @thornbill might know why those escaped whitespaces are used (not quite sure) but the error just looks like a max line length rule so you can fix it by making sure the line doesn't exceed a certain number of characters.

@dkanada  thank you, it's now solved. I wasn't sure what the relation of { ' ' } was with the link so couldn't understand the recommendation made. All sorted now, thanks!

**Review by nielsvanvelzen — APPROVED** [review]

**Review by thornbill — APPROVED** [review]

**Pull request merged**